api/migrations/versions/14d53fa0c9c8_.py

```diff
@@ -270,15 +270,15 @@ def upgrade():
     session.add(q1_nomination_capacity)
     session.commit()
 
-    nominator_information = Section(app_form_id, 'Nomination Information', """
+    nominator_information = Section(app_form_id, 'Nominator Information', """
```

Comment:
So unfortunately updating this previous migration won't work because it has already been deployed and applied to the staging database. You'll need to write updates in your new migration instead..

Reply:
Hi @avishkar58, yes sorry I did not think that through. I will revert all changes to that file. Do I then create a new migration and move the entire contents of the file over to it (as I did with the multiple head error)? Thanks for the patients :)